caliban_toolbox/aws_functions.py

```diff
@@ -91,7 +91,9 @@ def aws_upload_files(aws_folder, stage, upload_folder, pixel_only, label_only, r
     subfolders = re.split('/', aws_folder)
     subfolders = '__'.join(subfolders)
 
-    url_dict = {'pixel_only': pixel_only, 'label_only': label_only, 'rgb': rgb_mode}
+    # TODO: add back label_only once logic is working correctly in caliban
```

Comment:
Shouldn't this be added back now as the label_only issue was closed by Caliban #163?

Reply:
Yup! Done